core/src/main/java/org/mule/runtime/core/api/connectivity/ConnectivityTestingStrategy.java

```diff
@@ -21,7 +21,7 @@
 public interface ConnectivityTestingStrategy {
```

Comment:
Why is not this in mule-api instead of core? (same for other classes moved from tooling to core)